source/operator.rst

```diff
@@ -64,6 +69,58 @@ main subsystems of the cluster:
      - ``my-cluster-ssl-internal``
      - A secret with TLS certificate generated for *internal* communications, see :ref:`tls` for details
 
+   * - updateStrategy
+     - string
+     - ``SmartUpdate``
+     - A strategy Operator uses for :ref:`upgrades<operator-update>`
+
+.. _operator.upgradeoptions-section:
+
+`Upgrade Options Section <operator.html#operator-upgradeoptions-section>`_
+--------------------------------------------------------------------------------
+
+The ``upgradeOptions`` section in the `deploy/cr.yaml <https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/cr.yaml>`_ file contains various configuration options to control Percona XtraDB Cluster upgrades.
+
+.. tabularcolumns:: |p{2cm}|p{13.6cm}|
+
++-----------------+-------------------------------------------------------------------------------------------+
+|                 | .. _upgradeoptions-versionserviceendpoint:                                                |
+|                 |                                                                                           |
+| **Key**         | `upgradeOptions.versionServiceEndpoint                                                    |
+|                 | <operator.html#upgradeoptions-versionserviceendpoint>`_                                   |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Value**       | string                                                                                    |
++-----------------+-------------------------------------------------------------------------------------------+
+| **Example**     | ``https://check.percona.com/versions``                                                   |
```

Comment:
Let's change the example to match the default value, which should be the actual URL (Ask @delgod for the actual URL)

Reply:
actual URL is `https://check.percona.com/versions`
it works correctly https://check.percona.com/versions/v1/pxc-operator/1.5.0/latest?databaseVersion=5.7